charts/nobl9-agent/values.yaml

```
@@ -45,6 +45,7 @@ deployment:
   # -- Node selector
   nodeSelector: {}
   # disktype: ssd
+  tolerations: []
```

To make it consistent with other fields.

```suggestion
  # -- Tolerations
  tolerations: []
  # tolerations:
  #  - key: "key1"
  #    operator: "Equal"
  #    value: "value1"
  #    effect: "NoSchedule"
  #  - key: "key2"
  #    operator: "Exists"
  #    effect: "NoSchedule"
```